test/readonly/test.ts

```diff
@@ -29,10 +29,16 @@ describe('Aws read only Integration Testing', () => {
   it('installs the aws_account module', install(['aws_account']));
 
   it('inserts aws credentials', query(`
-    INSERT INTO aws_account (region, access_key_id, secret_access_key)
-    VALUES ('us-east-1', '${process.env.AWS_ACCESS_KEY_ID}', '${process.env.AWS_SECRET_ACCESS_KEY}')
+    INSERT INTO aws_credentials (access_key_id, secret_access_key)
+    VALUES ('${process.env.AWS_ACCESS_KEY_ID}', '${process.env.AWS_SECRET_ACCESS_KEY}')
   `, undefined, false));
 
+  it('syncs the regions', sync());
+
+  it('sets the default region', query(`
+    UPDATE aws_regions SET is_default = TRUE WHERE region = 'us-east-1';
+  `));
```

Comment:
technically all these tests could be changed to the following, right?
```suggestion
  it('sets the default region', query(`
    SELECT * FROM default_aws_region('us-east-1');
  `));
```

Reply:
Yes, they should be equivalent. Do you think it'd be better to make that change?